src/js/ko-components/view-models/orifice-calculator.coffee

```diff
@@ -3,9 +3,17 @@
 @OPL.KoComponents.ViewModels ||= {}
 
 class OPL.KoComponents.ViewModels.OrificeCalculator
+  COEFFICIENT_OF_DISCHARGE = 0.6
+  EXPANSION_FACTOR         = 1
+  BASE_TEMPERATURE         = 519.67
```

Comment:
Lets just add some notes here so we don't forget which unit is stored in this constant, please
Same thing applies to base pressure and absolute zero